coveo-stew/coveo_stew/offline_publish.py

```diff
@@ -132,6 +132,9 @@ def _store_setup_dependencies_in_wheelhouse(self, project: PythonProjectAPI = No
 
     def _store_dependencies_in_wheelhouse(self) -> None:
         """ Store the dependency wheels in the wheelhouse. """
+        # temporary mitigation of circular import

```

Comment:
Ça va être quoi ta solution long terme, i.e. comment est-ce que tu vas briser le cycle? :thinking:

Reply:
1. PythonProject deviendrait essentiellement une fancy dataclass pour gérer l'aspect I/O du pyproject.toml, au lieu d'évoluer en rammassi de toutes les fonctionalités.
2. e.g. Au lieu d'avoir install() dans le PythonProject, on a un module install.py qui fonctionne à partir d'une instance de PythonProject (repeat for all functionalities)